hax/hax/ha/event/event.py

```diff
@@ -16,11 +16,43 @@
 # opensource@seagate.com or cortx-questions@seagate.com.
 
 import logging
+from hax.ha.const import HEALTH_EVENT_SOURCES, NOT_DEFINED
+from hax.ha.utils import HaUtils
+from hax.util import ConsulUtil
+from cortx.utils.event_framework.health import HealthAttr
 
 
 class HaEvent():
-    def __init__(self):
+    def __init__(self,
+                 source: str,
+                 cluster_id: str,
+                 site_id: str,
+                 rack_id: str,
+                 storageset_id: str,
+                 node_id: str,
+                 resource_type: str,
+                 resource_id: str,
+                 resource_status: str,
+                 specific_info):
         logging.debug('Inside HaEvent')
+        self.payload = {
```

Comment:
Question: what's the purpose of having this payload attribute? Can't we simply create a  HealthEvent() directly? 
Also, we have this `event.set_specific_info()` to set anything specific in our child classes.

Reply:
Done, modified to create HealthEvent() directly.